### ISSUE TITLE
Can not build and run on MacOs

### ISSUE THREAD

**Issue by KanybekMomukeyev** [Drakirus/go-flutter-desktop-embedder#28]
Hi, thanks for sharing, I followed tutor for MacOs
but have following issue, what can cause this error?
```
[FATAL:flutter/fml/icu_util.cc(86)] Check failed: context->IsValid(). Must be able to initialize the ICU context. Tried: /opt/flutter/bin/cache/artifacts/engine/linux-x64/icudtl.dat
SIGABRT: abort
PC=0x7fff5d257b66 m=0 sigcode=0

goroutine 0 [idle]:
runtime: unknown pc 0x7fff5d257b66
stack: frame={sp:0x7ffeefbfda88, fp:0x0} stack=[0x7ffeefb804c8,0x7ffeefbff530)
00007ffeefbfd988:  0000000000000000  0000000000000000 
00007ffeefbfd998:  0000000000000000  0000000000000000 
00007ffeefbfd9a8:  0000000000000000  0000000000000000 
00007ffeefbfd9b8:  0000000000000000  00007fff95bee1a8 
00007ffeefbfd9c8:  00007ffeefbfda30  0000000000000002 
00007ffeefbfd9d8:  00007ffeefbfda38  00007ffeefbfda00 
00007ffeefbfd9e8:  0000040000000400  00007fff95bee240 
00007ffeefbfd9f8:  00007ffeefbfda60  0000000000000002 
00007ffeefbfda08:  00007ffeefbfda68  00007ffeefbfda30 
00007ffeefbfda18:  00007fff5d191d89  00007fff95bee240 
00007ffeefbfda28:  0000000000000000  00007ffeefbfda50 
00007ffeefbfda38:  00007fff5d192334  0000000000000003 
00007ffeefbfda48:  0000000000000000  00007ffeefbfdaa0 
00007ffeefbfda58:  00007fff5b069a70  00007fff95bee240 
00007ffeefbfda68:  00007fff95bedca0  00007fff5d192366 
00007ffeefbfda78:  0000000000000000  00007ffeefbfdac0 
00007ffeefbfda88: <00007fff5d422080  00007fff95c0f380 
00007ffeefbfda98:  00007ffeefbfdad8  000000000549e1c0 
00007ffeefbfdaa8:  00007ffeefbfe5e0  00007ffeefbfdb60 
00007ffeefbfdab8:  000000000000000a  00007ffeefbfdaf0 
00007ffeefbfdac8:  00007fff5d1b31ae  0000000000000400 
00007ffeefbfdad8:  00000000ffffffdf  ffffffffefbfdb40 
00007ffeefbfdae8:  00007ffeefbfdb58  00007ffeefbfdb40 
00007ffeefbfdaf8:  00000000042fd7fa  00007ffeefbfdb58 
00007ffeefbfdb08:  00000000000000c1  00000000000000b6 
00007ffeefbfdb18:  0000000005d780b0  08e027a4bbe90012 
00007ffeefbfdb28:  00007ffeefbfe840  00007ffeefbfdb58 
00007ffeefbfdb38:  00007ffeefbfdce8  00007ffeefbfdc90 
00007ffeefbfdb48:  00000000042fcf8e  0000000000000000 
00007ffeefbfdb58:  0000000005489058  0000000005488fd0 
00007ffeefbfdb68:  00007fff955029e0  0000000000000000 
00007ffeefbfdb78:  0000000000000000  0000000000000000 
runtime: unknown pc 0x7fff5d257b66
stack: frame={sp:0x7ffeefbfda88, fp:0x0} stack=[0x7ffeefb804c8,0x7ffeefbff530)
00007ffeefbfd988:  0000000000000000  0000000000000000 
00007ffeefbfd998:  0000000000000000  0000000000000000 
00007ffeefbfd9a8:  0000000000000000  0000000000000000 
00007ffeefbfd9b8:  0000000000000000  00007fff95bee1a8 
00007ffeefbfd9c8:  00007ffeefbfda30  0000000000000002 
00007ffeefbfd9d8:  00007ffeefbfda38  00007ffeefbfda00 
00007ffeefbfd9e8:  0000040000000400  00007fff95bee240 
00007ffeefbfd9f8:  00007ffeefbfda60  0000000000000002 
00007ffeefbfda08:  00007ffeefbfda68  00007ffeefbfda30 
00007ffeefbfda18:  00007fff5d191d89  00007fff95bee240 
00007ffeefbfda28:  0000000000000000  00007ffeefbfda50 
00007ffeefbfda38:  00007fff5d192334  0000000000000003 
00007ffeefbfda48:  0000000000000000  00007ffeefbfdaa0 
00007ffeefbfda58:  00007fff5b069a70  00007fff95bee240 
00007ffeefbfda68:  00007fff95bedca0  00007fff5d192366 
00007ffeefbfda78:  0000000000000000  00007ffeefbfdac0 
00007ffeefbfda88: <00007fff5d422080  00007fff95c0f380 
00007ffeefbfda98:  00007ffeefbfdad8  000000000549e1c0 
00007ffeefbfdaa8:  00007ffeefbfe5e0  00007ffeefbfdb60 
00007ffeefbfdab8:  000000000000000a  00007ffeefbfdaf0 
00007ffeefbfdac8:  00007fff5d1b31ae  0000000000000400 
00007ffeefbfdad8:  00000000ffffffdf  ffffffffefbfdb40 
00007ffeefbfdae8:  00007ffeefbfdb58  00007ffeefbfdb40 
00007ffeefbfdaf8:  00000000042fd7fa  00007ffeefbfdb58 
00007ffeefbfdb08:  00000000000000c1  00000000000000b6 
00007ffeefbfdb18:  0000000005d780b0  08e027a4bbe90012 
00007ffeefbfdb28:  00007ffeefbfe840  00007ffeefbfdb58 
00007ffeefbfdb38:  00007ffeefbfdce8  00007ffeefbfdc90 
00007ffeefbfdb48:  00000000042fcf8e  0000000000000000 
00007ffeefbfdb58:  0000000005489058  0000000005488fd0 
00007ffeefbfdb68:  00007fff955029e0  0000000000000000 
00007ffeefbfdb78:  0000000000000000  0000000000000000 

goroutine 1 [syscall]:
runtime.cgocall(0x40ea080, 0xc00009bbe8, 0x29)
	/usr/local/go/src/runtime/cgocall.go:128 +0x5e fp=0xc00009bbb0 sp=0xc00009bb78 pc=0x400582e
github.com/Drakirus/go-flutter-desktop-embedder/flutter._Cfunc_runFlutter(0x5c03d00, 0xc0000783c0, 0xc000060180, 0x5c4e810, 0x3, 0x0)
	_cgo_gotypes.go:327 +0x4d fp=0xc00009bbe8 sp=0xc00009bbb0 pc=0x40e34ad
github.com/Drakirus/go-flutter-desktop-embedder/flutter.(*EngineOpenGL).Run.func2(0x5c03d00, 0xc0000783c0, 0xc000060180, 0x5c4e810, 0x3, 0xc00000e0b0)
	/Users/user/Desktop/gocode/src/github.com/Drakirus/go-flutter-desktop-embedder/flutter/flutter.go:78 +0x164 fp=0xc00009bc28 sp=0xc00009bbe8 pc=0x40e4af4
github.com/Drakirus/go-flutter-desktop-embedder/flutter.(*EngineOpenGL).Run(0xc0000783c0, 0x5c03d00, 0xc0000841e0, 0x3, 0x3, 0xc00000e0a8)
	/Users/user/Desktop/gocode/src/github.com/Drakirus/go-flutter-desktop-embedder/flutter/flutter.go:78 +0x21e fp=0xc00009bcb8 sp=0xc00009bc28 pc=0x40e3d1e
github.com/Drakirus/go-flutter-desktop-embedder.runFlutter(0xc0000a8080, 0x320, 0x258, 0xc00001e200, 0x78, 0x4145879, 0x3c, 0x4146178, 0x3ff3333333333333, 0xc0000841e0, ...)
	/Users/user/Desktop/gocode/src/github.com/Drakirus/go-flutter-desktop-embedder/gutter.go:236 +0x22b fp=0xc00009bd68 sp=0xc00009bcb8 pc=0x40e645b
github.com/Drakirus/go-flutter-desktop-embedder.Run(0xc0000a8000, 0xa, 0x10, 0x0, 0x0)
	/Users/user/Desktop/gocode/src/github.com/Drakirus/go-flutter-desktop-embedder/gutter.go:43 +0x2ee fp=0xc00009bec0 sp=0xc00009bd68 pc=0x40e57de
main.main()
	/Users/user/Desktop/Research/go-flutter-desktop-embedder/example/simpleDemo/main.go:37 +0x26e fp=0xc00009bf98 sp=0xc00009bec0 pc=0x40e966e
runtime.main()
	/usr/local/go/src/runtime/proc.go:201 +0x207 fp=0xc00009bfe0 sp=0xc00009bf98 pc=0x402d2f7
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1333 +0x1 fp=0xc00009bfe8 sp=0xc00009bfe0 pc=0x40557c1

rax    0x0
rbx    0x7fff95c0f380
rcx    0x7ffeefbfda88
rdx    0x0
rdi    0x307
rsi    0x6
rbp    0x7ffeefbfdac0
rsp    0x7ffeefbfda88
r8     0x7fff95bee048
r9     0x40
r10    0x0
r11    0x206
r12    0x307
r13    0x7ffeefbfe5e0
r14    0x6
r15    0x2d
rip    0x7fff5d257b66
rflags 0x206
cs     0x7
fs     0x0
gs     0x0
```

**Comment by Drakirus** [comment]
@zephylac worked on the installation procedure.
Checkout
-  [icudtl.dat config](https://github.com/Drakirus/go-flutter-desktop-embedder/blob/8186aaf1a3faeb52a2259718c205a740bcb24a04/example/simpleDemo/main.go#L32-L35)
 - The updated readme`go run engineDownloader.go`

**Comment by Drakirus** [comment]
Should be fixed with the PRs made by @zephylac,  #33 #31 #29.
Closing the issue, feel free to reopen, comment!
